packages/xstate-react/src/useMachine.ts

```diff
@@ -133,7 +133,7 @@ export function useMachine<
     ...interpreterOptions
   } = options;
 
-  const service = useConstant(() => {
+  const customMachine = useConstant(() => {
```

Comment:
nit: custom sounds off here to me, maybe one of final/created/merged? or just `machine` + renaming the top-level parameter to `inputMachine`?